q/gradle-quickstart.sh

```diff
@@ -116,7 +116,7 @@ ext {
     scalaBinaryVersion = '${scalaBinaryVersion}'
     slf4jVersion = '1.7.32'
```

Comment:
```suggestion
    slf4jVersion = '1.7.36'
```
if we want to fix these in general.